### PR TITLE
Plumbing for CADMIN attribute updates from fabric-admin to fabric-bridge

### PR DESCRIPTION
**Pull request by tehampson**
usage of fabric-admin calling `AdminCommissioningAttributeChanged` will be done in a follow up PR. This allows this PR to focus on plumbing of data from fabric-admin to fabric-bridge

This code is not yet heavily tested as fabric-admin still needs to subscribe to the attribute and properly populate the values. This will be completed in a follow up and any issues identified in the plumbing in this PR will be addressed in a follow up.

### CODE REVIEW

File: examples/fabric-admin/rpc/RpcClient.h [hunk 1]
```diff
@@ -74,3 +74,14 @@ CHIP_ERROR RemoveSynchronizedDevice(chip::NodeId nodeId);
...
+/**
+ * @brief CADMIN attribute has changed of one of the bridged devices that was previously added.
+ *
+ * @param data information regarding change in AdministratorCommissioning attributes
+ * @return CHIP_ERROR An error code indicating the success or failure of the operation.
+ * - CHIP_NO_ERROR: The RPC command was successfully processed.
+ * - CHIP_ERROR_BUSY: Another operation is currently in progress.
+ * - CHIP_ERROR_INTERNAL: An internal error occurred while activating the RPC call.
+ */
+CHIP_ERROR AdminCommissioningAttributeChanged(const chip_rpc_AdministratorCommissioningChanged & data);
```
Comment (yufengwangca) [inline examples/fabric-admin/rpc/RpcClient.h:87]: When and where this function get called?

File: examples/fabric-bridge-app/linux/RpcServer.cpp [hunk 2]
```diff
@@ -160,6 +162,52 @@ pw::Status FabricBridge::ActiveChanged(const chip_rpc_KeepActiveChanged & reques
...
+        static_cast<chip::app::Clusters::AdministratorCommissioning::CommissioningWindowStatusEnum>(request.window_status);
+    if (request.has_opener_fabric_index)
+    {
+        VerifyOrReturnValue(request.opener_fabric_index >= chip::kMinValidFabricIndex, pw::Status::InvalidArgument());
+        VerifyOrReturnValue(request.opener_fabric_index <= chip::kMaxValidFabricIndex, pw::Status::InvalidArgument());
+        adminCommissioningAttributes.openerFabricIndex = static_cast<FabricIndex>(request.opener_fabric_index);
+    }
+    else
+    {
+        adminCommissioningAttributes.openerFabricIndex.reset();
```
Comment (yufengwangca) [inline examples/fabric-bridge-app/linux/RpcServer.cpp:194]: it looks openerFabricIndex field hasn't been modified before calling reset() in the scenario where request.has_opener_fabric_index is false.

File: examples/fabric-bridge-app/linux/RpcServer.cpp [hunk 3]
```diff
@@ -160,6 +162,52 @@ pw::Status FabricBridge::ActiveChanged(const chip_rpc_KeepActiveChanged & reques
...
+    }
+
+    if (request.has_opener_vendor_id)
+    {
+        VerifyOrReturnValue(request.opener_vendor_id != chip::VendorId::NotSpecified, pw::Status::InvalidArgument());
+        adminCommissioningAttributes.openerVendorId = static_cast<chip::VendorId>(request.opener_vendor_id);
+    }
+    else
+    {
+        adminCommissioningAttributes.openerVendorId.reset();
```
Comment (yufengwangca) [inline examples/fabric-bridge-app/linux/RpcServer.cpp:204]: Same as above, openerVendorId field hasn't been modified before calling reset() in the scenario where request.has_opener_vendor_id is false.

File: examples/fabric-admin/rpc/RpcClient.h (hunk 1 above)
Comment (tehampson) [reply examples/fabric-admin/rpc/RpcClient.h:87]: In a follow up PR. I am trying to keep reviews simple and easy to review

File: examples/fabric-bridge-app/linux/RpcServer.cpp (hunk 2 above)
Comment (tehampson) [reply examples/fabric-bridge-app/linux/RpcServer.cpp:194]: It can have a previous value from being called before

**Review by yufengwangca — APPROVED** [review]

File: examples/fabric-admin/rpc/RpcClient.h (hunk 1 above)
Comment (tehampson) [reply examples/fabric-admin/rpc/RpcClient.h:87]: https://github.com/project-chip/connectedhomeip/pull/35239

**Review by saurabhst — APPROVED** [review]

**Pull request merged**